m2plots/time-series/latestyear_spaghetti_ncaregions.py

```diff
@@ -149,7 +149,14 @@
 
 prec:
   variablename: 'PRECTOT'
-  varlongname: 'Precipitation'
+  varlongname: 'Modeled Precipitation'
```

Comment:
Do we want to do this, or...

```suggestion
  varlongname: 'Precipitation (atmosphere)'
```

Reply:
After giving it some thought, I am afraid users will get more confused with the terms "atmosphere" and "land", as though the precipitation stays in the atmosphere and doesn't get deposited out.